Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -299,11 +299,30 @@ function Wait-ServiceFabricApplicationUpgradeAction
     }
 
     $upgradeStatusValidator = { param($upgradeStatus) return !($upgradeStatus.UpgradeState -eq "RollingBackCompleted" -or $upgradeStatus.UpgradeState -eq "RollingForwardCompleted") }
+
+    $exceptionRetryEvaluator = {
```

Comment:
Do we need to handle this for other tasks i.e. Service Fabric PowerSHell and Compose deploy

Reply:
If you think there can be a possible use case (for ServiceFabricPowerShell or ComposeDeployTask) where user can use AAD backed cluster and try to perform an operation which can take over 1 hour i.e. default access token expiry time, then we can add support for the same.

Reply:
Not easy to do this in powershell task and don't know whether people are facing this problem in compose task. @kuvinodms can you please check this error code in telemetry to see how many folks hitting it.